src/pybind/mgr/dashboard/frontend/src/app/ceph/nfs/nfs-form/nfs-form.component.ts

```diff
@@ -617,12 +648,14 @@ export class NfsFormComponent extends CdForm implements OnInit {
 
   private buildRequest() {
     const requestModel: any = _.cloneDeep(this.nfsForm.value);
-
+    const path = this.nfsForm.get('path').value;
     if (this.isEdit) {
       requestModel.export_id = _.parseInt(this.export_id);
       if (requestModel.fsal.name === SUPPORTED_FSAL.RGW) {
         requestModel.fsal.user_id = this.nfsForm.getValue('fsal').user_id;
-        requestModel.path = this.nfsForm.getValue('path');
+        requestModel.path = path;
+      } else {
+        requestModel.path = path;
```

Comment:
it looks like we don't even need this else block. you could just simply move it below the 
`requestModel.export_id = _.parseInt(this.export_id);` since the path is needed for both rgw and nfs.